node/config/def.go

```diff
@@ -91,6 +91,15 @@ type MinerFeeConfig struct {
 type MinerAddressConfig struct {
 	PreCommitControl []string
 	CommitControl    []string
+	TerminateControl []string
+
+	// DisableOwnerFallback disables usage of the owner address for messages
+	// sent automatically
+	DisableOwnerFallback bool
+	// DisableWorkerFallback disables usage of the worker address for messages
+	// sent automatically if control addresses are configured, even if those
```

Comment:
// DisableWorkerFallback disables usage of the worker address for messages
// sent automatically, **if** control addresses are configured.
// A control address that doesn't have enough funds will still be chosen over the worker address if this flag is set.

Reply:
(found the comment confusing)